doc/developer-guide/api/functions/TSHttpTxnClientProtocolGet.en.rst

```diff
@@ -0,0 +1,34 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed
+   with this work for additional information regarding copyright
+   ownership.  The ASF licenses this file to you under the Apache
+   License, Version 2.0 (the "License"); you may not use this file
+   except in compliance with the License.  You may obtain a copy of
+   the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   implied.  See the License for the specific language governing
+   permissions and limitations under the License.
+
+.. include:: ../../../common.defs
+
+.. default-domain:: c
+
+TSHttpTxnClientProtocolGet
+*********************
+
+Gets the protocol string (http, http/2) of a specified transaction. 
+
+Synopsis
+========
+
+`#include <ts/ts.h>`
+
+.. function:: const char * TSHttpTxnClientProtocolGet(TSHttpTxn txnp)
+
+Description
+===========
```

Comment:
I am not sure if I am the right person to mess with docs because I am relatively new to the ATS practices. However I feel like TSHttpTxn* functions can be mashed together, same with TSHttpSsn*, same with TSMimeHdr*, TSHttpHdr* and so on. I am basing this off the history of my needs and my intuition that a user of the interface would want to see in one place ALL they can do with a Transaction, Session, etc... This will reduce the number of files in this folder significantly.